src/index.coffee

```diff
@@ -12,6 +12,11 @@ module.exports = class StaticHandlebarsCompiler
   constructor: (@config) ->
     @outputDirectory = @config?.plugins?.staticHandlebars?.outputDirectory || 'public'
 
+    # Load additional configuration / context from a project-specific file
+    includeFile = @config?.plugins?.staticHandlebars?.includeFile || 'staticHandlebarsExtras.js'
```

Comment:
@Faldrian designating a default filename seems overly prescriptive. Would you mind removing the default here?